streamer/input_configuration.py

```diff
@@ -37,7 +37,16 @@ class InputType(enum.Enum):
   The device path should be given in the name field.  For example, on Linux,
   this might be /dev/video0.
 
-  Does not support media_type of 'text'.
+  Only supports media_type of 'video'.
```

Comment:
Are you sure this change was correct?  I suspect you could have a webcam with a built-in mic and get audio from it.

Reply:
I didn't know that webcams may contain mics O_O.
But it think it depends more on the input device we are using for this platform. for example `video4linux2` can't be used an audio input device, while `dshow` would work for both audio and video, so we may have an input like this on Windows:
```yaml
inputs:
    - name: "video=Integrated Camera"
      input_type: microphone
      media_type: video
```
and it would still work as if it were a webcam video input and not a microphone.
and we may use webcam input type to capture a microphone device too
```yaml
inputs:
    - name: "audio=Microphone Name"
      input_type: webcam
      media_type: audio
```
this happens on Windows and i think with the `avfoundation` on macOS as well.
Although a microphone input device a using webcam input type might work, i don't think this is how the input config should be for it. it should be using a microphone input type when capturing a mic device.

I believe in the case you mentioned, a webcam device that has a built-in mic, the mic would still show up as an audio input device that our args we provide for microphone devices(`PulseAudio`, `AVFoundation`, `dshow`) would still work for, so why not declare this input device as a mic in the first place.